force-app/main/default/triggers/MyOpportunityTrigger.trigger

```diff
@@ -0,0 +1,60 @@
+trigger MyOpportunityTrigger on Opportunity (before update, before delete, after update) {
+
+    // Question #5: Prevent update if Amount < 5000
+    if(Trigger.isBefore && Trigger.isUpdate){
+        for(Opportunity opp : Trigger.new){
+            if(opp.Amount < 5000){
```

Comment:
```suggestion
            if(opp.Amount <= 5000){
```
The requirements say GREATER than 5000, so I would assume 5000 also should cause an error.